core/src/main/scala/pureconfig/CollectionReaders.scala

```diff
@@ -23,6 +23,20 @@ trait CollectionReaders {
       }
     }
 
+  implicit def eitherReader[A, B](implicit convA: ConfigReader[A], convB: ConfigReader[B]): ConfigReader[Either[A, B]] =
+    new ConfigReader[Either[A, B]] {
+      override def from(cur: ConfigCursor): ConfigReader.Result[Either[A, B]] = {
+        convA.from(cur) match {
```

Comment:
Should it be right biased? That we first try if we can read as right value and only after fall back to left one..

Reply:
Fair point! Addressed this in a change.